src/Avalonia.Visuals/Visual.cs

```diff
@@ -68,6 +68,12 @@ public class Visual : StyledElement, IVisual
         public static readonly StyledProperty<IBrush?> OpacityMaskProperty =
             AvaloniaProperty.Register<Visual, IBrush?>(nameof(OpacityMask));
 
+        /// <summary>
+        /// Defines the <see cref="IsMirrorTransform"/> property.
+        /// </summary>
+        public static readonly DirectProperty<Visual, bool> IsMirrorTransformProperty =
```

Comment:
```suggestion
        public static readonly DirectProperty<Visual, bool> HasMirrorTransformProperty =
```

Reply:
In correct English you are right, I just barely saw the `Has` property, I will change it.

Reply:
Well, hate to comment again. However, the mirror transform is constant and always known. The transform itself is never set and what this property does is tell if the control is mirrored using the constant transform. So in my mind 'IsMirrored' makes more sense.

Reply:
@Gillibald